### PR TITLE
multi-route API

### PR DESCRIPTION
**Pull request by maksimlikharev**
conversion steps + test

### CODE REVIEW

File: initial-config/static/apis/petstore-oauth/config.json [hunk 1]
```diff
@@ -2,8 +2,15 @@
     "api": {
         "upstream_url": "http://petstore.swagger.io/v2",
         "request_path": "/petstore-oauth",
-        "preserve_host": false,
-        "strip_request_path": true,
+        "routes": [
+          {
+            "paths": [
+              "/petstore"
```
Comment (DonMartin76) [inline initial-config/static/apis/petstore-oauth/config.json]: Wrong path

File: initial-config/static/apis/petstore-oauth/config.json [hunk 2]
```diff
@@ -2,8 +2,15 @@
     "api": {
         "upstream_url": "http://petstore.swagger.io/v2",
         "request_path": "/petstore-oauth",
```
Comment (DonMartin76) [inline initial-config/static/apis/petstore-oauth/config.json]: Should this stay in?

**Review by DonMartin76 — CHANGES_REQUESTED** [review]

**Comment by DonMartin76** [comment]
You wrote tests for the transformation, which I like, but we don't run them during the build yet. We should think about how we can get that in as well.

**Comment by DonMartin76** [comment]
Please merge latest changes from master as well.

**Comment by maksimlikharev** [comment]
different teams use different approach, but in our UI projects, normal unit tests are part of the build, and integration tests are just a separate project/repo. I can add all necessary deps/hooks into the build. On a second thought, you might want to add linters there as well.

**Pull request closed without merging**